src/components/loginSignup/loginsignup.css

```diff
@@ -0,0 +1,202 @@
+.login {
+  /* width: 20%;
+  position: absolute;
+  top: 30%;
+  left: 43%;
+  /* border: 1px solid; */
+}
+
+.logoDetail {
+  position: absolute;
+  width: 45%;
+  height: 100vh;
+  left: 0px;
+  top: 0px;
+  background: linear-gradient(180deg, #f0a500 0%, #cf7500 100%);
+  opacity: 0.8;
+}
+
+.backgroundImg {
+  width: 100%;
+  height: 100%;
+}
+
+.logoImg {
+  position: absolute;
+  width: 43%;
+  height: 15%;
+  left: 27%;
+  top: 40%;
+  background: #d6d6d6;
+  box-shadow: 0px 22.4801px 22.4801px rgba(0, 0, 0, 0.25);
+}
+
+.formDetails {
+  position: absolute;
+  width: 55%;
+  height: 100vh;
+  left: 45%;
+  top: 0px;
+  background: #ffffff;
+  box-shadow: 0px 4.89691px 20.4038px rgba(0, 0, 0, 0.25);
+}
+
+.signupBox {
+  width: 50%;
+  background-color: rgb(181, 181, 181, 0.1);
+  position: relative;
+  left: 25%;
+  top: 10%;
+  height: 80%;
+  position: absolute;
+
+  /* animation: signupfade 5s ease-in; */
+}
+
+/* @keyframes signpfade {
+  0% {
+    width: 0%;
+  }
+
+  100% {
+    width: 50%;
+  }
+} */
+
+.formBox {
+  width: 50%;
+  background-color: rgb(181, 181, 181, 0.1);
+  position: relative;
+  left: 25%;
+  top: 10%;
+  height: 80%;
+  position: absolute;
+}
+
+.signin {
+  text-align: center;
+  position: absolute;
+  height: 69.2px;
+  font-family: Roboto;
+  font-size: 34px;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
```

Comment:
Maybe use [font shorthand](https://developer.mozilla.org/en-US/docs/Web/CSS/font).

Reply:
ille..its not understandable. later will do that

Reply:
Okay.
But is font-stretch, font-style necessary? Aren't they by default normal?

Reply:
i dont know ..i copied wat rihan wrote